src/controller/busControllers.js

```diff
@@ -87,6 +92,107 @@ class busController {
             message: res.__("Deleted successfully")
         })
     }
+    static async assignDriver(req,res){
+       
+        try{
+            const user = await Model.User.findOne({
+                where: { email:req.body.email},
+              });
+          
+           
+            const updated = await Model.Bus.update({ userId:user.id },
+                        { where: {id:req.params.id } }
+                     );
+                     if (updated) {
+                            const bus= await Model.Bus.findOne({
+                                 where: { id:req.params.id },
+                                 include:[{
+                                     model:Model.User,
+                                     as:'driver',
+                                     attributes:["id","name","email"]
+                                  }]
+                                 });
+                                 console.log(bus)
+                            const Options ={
+                                userEmail:req.body.email,
+                                subject:'Phantom assignment',
+                                message:message.assignMessage(user.name,bus.plate)
+                                
+                              }
+                              emails.sendEmail(Options) 
+                            return res.status(200).json({ 
+                               bus 
+                            });
+                        }
+            
+        }catch (error) {
+            return res.status(500).json({ message: error })
+        }
+    }
+
+    static async unassignDriver(req,res){
+        try {
+            
+            const updated = await Model.Bus.update({ userId:null },
+                { where: {id:req.params.id } }
+             );
+             if (updated) {
+                    const bus= await Model.Bus.findOne({
+                         where: { id:req.params.id },
+                        
+                         });
+
+                    const Options ={
+                        userEmail:req.body.email,
+                        subject:'Phantom Unassignment',
+                        message:message.unassignMessage(req.user.name,bus.plate)
+                        
+                      }
+                      emails.sendEmail(Options) 
+                    return res.status(200).json({ 
+                       bus 
+                    });
```

Comment:
This section is similar to the one in `assignDriver` with a few differences. Can you refactor this into a re-usable function?